build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.4.2'
+    classpath 'com.android.tools.build:gradle:3.5.0'
```

Comment:
Do we really need this update?

Reply:
This doesn't appear resolved yet. Please downgrade Gradle per https://groups.google.com/forum/#!topic/oppia-android-dev/a17n7RGRnCA.

Reply:
okay